crates/forge/tests/cli/coverage.rs

```diff
@@ -4,3 +4,8 @@ forgetest!(basic_coverage, |_prj: TestProject, mut cmd: TestCommand| {
     cmd.args(["coverage"]);
     cmd.assert_success();
 });
+
+forgetest!(report_file_coverage, |_prj: TestProject, mut cmd: TestCommand| {
+    cmd.arg("coverage").args(["--report".to_string(), "lcov".to_string(), "--report-file".to_string(), "/path/to/lcov.info".to_string()]);
```

Comment:
for the path here, I think we could do something like `_prj.root().join('lcov.info')`—this way we avoid passing a somewhat ambiguous path that might cause the test to be spurious

Reply:
I'm unsure about where to add `_prj.root().join('lcov.info')` , I'm not familiar with how `forgetest` works yet :(

Reply:
for arguments that expect an output path, like `"/path/to/lcov.info".to_string()` replace with `prj.root().join('lcov.info')`

so that we ensure the file is written inside the test project's folder